docs/reporters/AntennaAttributionDocumentReporter.md

```diff
@@ -0,0 +1,49 @@
+# AntennaAttributionDocumentReporter
+
+The AntennaAttributionDocumentReporter generates an attribution document in PDF format by leveraging the [Eclipse 
+Antenna][1] project. The attribution document can be customized by providing a template bundle JAR, or a directory with 
+template PDF files. 
+There is a guide in the [Antenna project documentation][2] which describes how such a template bundle JAR file for the 
+customization has to be created.
+
+In addition to using a template bundle JAR, there is an alternative way to customize the attribution document. For this, 
+a directory with template PDF files has to be provided. The PDF file names have to match the following:
+ * `cover.pdf`: Template PDF file for the cover page.
```

Comment:
Please example of the PDF files to `docs/examples` directory so users can quickly try out this reporter with file that are known to work.

Reply:
I believe we should put the PDFs in a "AntennaAttributionDocument" sub-directory then, to not interleave them with the other (configuration) examples files. This is esp. also as probably more AntennaAttributionDocument-specific files, like fonts, are about to come.